core/domain/draft_upgrade_services.py

```
@@ -312,6 +363,63 @@ def _convert_html_in_draft_change_list(
+    ) -> List[exp_domain.ExplorationChange]:
+        """Converts draft change list from state version 52 to 53. State
+        version 53 ensures that the content html does not contains links
+        that use a schema other than 'https'.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+
+        for exp_change in draft_change_list:
+            if not exp_change.cmd == exp_domain.CMD_EDIT_STATE_PROPERTY:
+                continue
+            # The change object has the key 'new_value' only if the
+            # cmd is 'CMD_EDIT_STATE_PROPERTY' or
+            # 'CMD_EDIT_EXPLORATION_PROPERTY'.
+            new_value: AllowedDraftChangeListTypes = exp_change.new_value
+            if exp_change.property_name == exp_domain.STATE_PROPERTY_CONTENT:
+                # Here we use cast because this 'if' condition forces
+                # change to have type EditExpStatePropertyContentCmd.
+                edit_content_property_cmd = cast(
+                    exp_domain.EditExpStatePropertyContentCmd,
+                    exp_change
+                )
+                new_value = edit_content_property_cmd.new_value
+                html_content = new_value['html']
+
+                cls._invalidate_drafts_with_invalid_links(html_content)
+
+            elif exp_change.property_name == (
+                exp_domain.STATE_PROPERTY_WRITTEN_TRANSLATIONS):
+                assert isinstance(exp_change.new_value, dict)
+
+                written_translations = exp_change.new_value
+                for translations in (
+                    written_translations['translations_mapping'].values()
+                ):
+                    for written_translation in translations.values():
+                        if isinstance(
```

If the instance is of type `list` simply remove the draft, we do not have translations present inside the list. I recently had a discussion with @vojtechjelinek regarding this part. We also have a validation regarding this on our schema_utils file.
**Edit** Firstly check if the `written_translation['data_format'] == 'html'`, if that is the case then we should simply raise the error regarding the list.

---

Done.